app.js

```diff
@@ -0,0 +1,34 @@
+var express = require('express');
```

Comment:
this file has linting errors via xo...as you are not using ecma 2015 but the version of xo you are using is for ecma 2015.